### ISSUE TITLE
Update the guides to show how users can use Liberty Tools to work with the guides

### ISSUE THREAD

**Issue by yeekangc** [OpenLiberty/guides-common#855]
Liberty Tools enable developers to work with Liberty from different IDEs like Eclipse IDE, IntelliJ IDEA and VS Code.

Update the guides to show users how they can use Liberty Tools to work through the guides.  e.g., update the Getting Started section (template) to cover how Liberty Tools can be used.

Cc @cherylking @scottkurz @TrevCraw @kathrynkodama @gkwan-ibm

**Comment by yeekangc** [comment]
FYI, @lauracowen @dmuelle @kinueng.

**Comment by lauracowen** [comment]
I don't think we should just add more instructions to the existing guides. It makes them longer and the guides are intended to be agnostic as to how the tools used. And we shouldn't make readers plough through irrelevant instructions.

Maybe could add alternative instructions (eg like offering different platforms or maven/gradle instructions, maybe?)? 

I agree that the Liberty Tools could be showcased a little in the Getting Started guide, though, as that's meant to be an introductory guide.

**Comment by gkwan-ibm** [comment]
Opened an [issue](https://github.ibm.com/skills-network/labs/issues/1683) to see whether cloud-hosted lab can run LT or not
